app/javascript/components/Results/ResultGrid/ResultGrid.tsx

```diff
@@ -68,20 +72,20 @@ export const ResultGrid = ({ result, affiliate, query, position, vertical }: Res
           {result.publishedAt   && <span className='published-date'>{result.publishedAt}</span>}
           <div className='result-title'>
             <h2 className='result-title-label'>
-              <a href={result.url}
-                className='result-title-link'
-                onClick={() => clickTracking(affiliate, module, query, position, result.url, vertical)}>
+              <ResultTitle 
```

Comment:
`<ResultTitle>` component can be imported at other modules like Jobs, Video, RSS etc for clickTracking implementation
@lorawoodford